src/SingleProject/Resizetizer/src/Utils.cs

```diff
@@ -15,17 +14,20 @@ internal class Utils
 		public static bool IsValidResourceFilename(string filename)
 			=> rxResourceFilenameValidation.IsMatch(Path.GetFileNameWithoutExtension(filename));
 
-		public static Color? ParseColorString(string tint)
+		public static SKColor? ParseColorString(string tint)
 		{
 			if (string.IsNullOrEmpty(tint))
 				return null;
 
-			if (int.TryParse(tint.Trim('#'), NumberStyles.HexNumber, CultureInfo.InvariantCulture, out var value))
-				return Color.FromArgb(value);
+			if (SKColor.TryParse(tint, out var color))
+			{
+				return color;
+			}
 
 			try
 			{
-				return Color.FromName(tint);
+				var drawing = System.Drawing.Color.FromName(tint);
```

Comment:
This is the last remaining use of `System.Drawing`. Does `SkiaSharp` have an equivalent? We could remove `System.Drawing.Common` if there is.

Reply:
@mattleibow ?

Reply:
Right now, SkiaSharp does not have this feature. However, since this is "parsing colors" we can do a lookup on field names for now on the `SKColors` type? Like `typeof(SKColors).GetFields(tint).GetValue` or something. Might need to check casing...

Reply:
I ported this over to support `SKColor` / `SKColors`: 

https://github.com/dotnet/runtime/blob/776dd1d3318760bf2328f66a544ed377d78b2099/src/libraries/Common/src/System/Drawing/ColorTable.cs